### ISSUE TITLE
[PNI QA] Author photos look a little pixelated 

### ISSUE THREAD

**Issue by beccaklam** [mozilla/foundation.mozilla.org#9608]
### Context: 
The author profiles at the bottom of article pages look pixelated on any device and on any browser (chrome, safari, firefox):

<img src="https://images.zenhubusercontent.com/57acd65ce40e5714b16d4215/613371e5-0efc-45cf-a2d1-44e08bca0007" width=300 />

### Acceptance Criteria:
- [ ] As user navigating the PNI article page using a device (mobile or desktop) with retina display, I can see the author profile images not pixelated 

### Dev note: 
- It's recommended to use an image size that's at least 2 times the display size

**Comment by tbrlpld** [comment]
That should be based on the originally uploaded image. We need the profile images to be uploaded with better quality if we want to display them larger.

**Comment by beccaklam** [comment]
@tbrlpld So I uploaded a 3.4MB photo (should be more than sufficient for the size of the author profile) but the resulting author profile is still pixelated?
<img width="1292" alt="image" src="https://user-images.githubusercontent.com/20977376/200052519-796604f5-c5f2-4d48-92f2-5223cf8bf9ee.png">
![IMG_CB85931994F6-1](https://user-images.githubusercontent.com/20977376/200052558-9a4fd4f2-8297-46cf-bb50-e28c56c54b45.jpeg)

**Comment by tbrlpld** [comment]
Thanks @beccaklam Looks like we need to investigate this.

**Comment by tbrlpld** [comment]
@beccaklam Can you confirm the device you are taking these screenshots on? Is that an iPhone?

**Comment by beccaklam** [comment]
@tbrlpld I don't remember where exactly the above screen shot is from but the author photos are pixelated on both my macbook and iPhone. Does that help?

**Comment by tbrlpld** [comment]
Thanks @beccaklam it does. I was just trying to confirm that you were using a high-resolution display (e.g. retina, which iPhone and MacBook have). I think it should not have been pixelated on normal resolution displays.

**Comment by danielfmiranda** [comment]
Just tested this on staging and can confirm that it is working as expected. Ready for prod!

**Comment by tbrlpld** [comment]
Checked on staging and is looking good, given that the source image has a good resolution. (We just did expedited QA to unblock a prod fix for @mtdenton)

@beccaklam Please double check this meets your requirements. Fixes would be in a new ticket.

**Comment by beccaklam** [comment]
This looks good to me! 👍 Thank you!

**Comment by cdanfon** [comment]
Hey @tbrlpld @danielfmiranda @fessehaye can I check with you whether this fix would also affect the issue raised [here](https://torchbox.slack.com/archives/C98GW3P7X/p1669981734915849) by a DFL colleague? For reference in that issue she's referring to the new streamfield block created for Data Futures Lab.

**Comment by tbrlpld** [comment]
@cdanfon No, the change won't have any effect outside of the Buyer's Guide article page because the authors are rendered directly in the page template. 

It might be a good idea to create more template fragments / components for these recurring patterns.

**Comment by cdanfon** [comment]
Thanks for confirming @tbrlpld, are you happy to create a ticket with the 'Maintenance' label for the template fragments/components please? Sounds like this could save us some time in the future
